src/containers/shared/components/Transaction/PaymentChannelClaim/test/PaymentChannelClaimDescription.test.tsx

```diff
@@ -0,0 +1,70 @@
+import i18n from '../../../../../../i18nTestConfig.en-US'
+
+import { createDescriptionWrapperFactory } from '../../test/createWrapperFactory'
+import mockPaymentChannelClaim from './mock_data/PaymentChannelClaim.json'
+import mockPaymentChannelClaimClosed from './mock_data/PaymentChannelClaimClosed.json'
+import mockPaymentChannelClaimCloseDenied from './mock_data/PaymentChannelClaimCloseDenied.json'
+import mockPaymentChannelClaimWithDestinationTag from './mock_data/PaymentChannelClaimWithDestinationTag.json'
+import { Description } from '../Description'
+
+const createWrapper = createDescriptionWrapperFactory(Description, i18n)
+
+describe('PaymentChannelClaim: Description', () => {
+  it('renders a claim', () => {
+    const wrapper = createWrapper(mockPaymentChannelClaim)
+    expect(wrapper.find('[data-test="account-line"]')).toHaveText(
+      `The transaction was initiated by rK6g2UYc4GpQH8DYdPG7wywyQbxkJpQTTN`,
+    )
+    expect(wrapper.find('[data-test="channel-line"]')).toHaveText(
+      `It will update the payment channel 50107651E7163E294CE0EAD8A20BF7CC046304480FCC9C74A49FFAB3F46FB98E`,
+    )
+    expect(wrapper.find('[data-test="balance-line"]')).toHaveText(
+      `The channel balance claimed is \uE90049.65716XRP (increased_by \uE9000.01XRP)`,
```

Comment:
Why is there an underscore in `increased_by` here?

(If it's referencing the translation - should the translations also be moved from snake_case to camelCase?)

Reply:
Great catch.  It was referencing a translation that was never populated.  The bug existed in prod.  This is one of the reasons I updated many of the tests to check the english values instead of checking to see if a key was used.  When making these tests I was copying what was the values were in prod and missed this one.